tests/python/pants_test/backend/python/tasks/test_python_distribution_integration.py

```diff
@@ -158,6 +156,7 @@ def test_pants_resolves_local_dists_for_current_platform_only(self):
       pex = os.path.join(tmp_dir, 'main.pex')
       pants_ini_config = {
         'python-setup': {
+          # TODO: explain why we need 'this-platform-does_not-exist' here.
```

Comment:
Can you knock this out now?

Reply:
Took a stab at answering in d5f00f9.

Reply:
*(it was left as a todo because I didn't know the answer)*